app/Console/Commands/GenerateResourse.php

```diff
@@ -39,14 +39,13 @@ public function __construct()
      */
     public function handle()
     {
-        $table_name = strtolower($this->argument('controllerName'));
-        shell_exec("/usr/bin/php artisan make:controller ".$this->argument('controllerName'));
-        if(count($this->option('schema'))!= 0)
-            shell_exec("/usr/bin/php artisan make:migration:schema create_".$table_name."s_table --schema=\"".$this->option('schema')."\"");
-        else
-            shell_exec("/usr/bin/php artisan make:migration create_".$table_name."_table");
-        shell_exec("/usr/bin/php artisan make:request ".$this->argument('controllerName'));
-        shell_exec("/usr/bin/php artisan make:form Forms/".$this->argument('controllerName'));
+        $table_name = strtolower($this->argument('tableName'));
+        $form_name = $this->argument('tableName');
+        $form_name[0]=strtoupper($form_name[0]);
+        shell_exec("/usr/bin/php artisan make:controller Admin/".$this->argument('controllerName').'Controller');
+        shell_exec("/usr/bin/php artisan make:migration create_".$table_name."_table");
+        shell_exec("/usr/bin/php artisan make:request Admin/".$this->argument('controllerName').'Request');
+        shell_exec("/usr/bin/php artisan make:form Forms/Admin/".$form_name.'Form');
         shell_exec("/usr/bin/php artisan migrate");
```

Comment:
This may be unexpected for users, so let the users decide when to run the migration.

Reply:
Also if the table argument is empty, can you guess the table name. Something like below.

```php
$table = $this->argument('tableName') ?? Str::snake(Str::plural($this->argument('controllerName')));
```